src/git-duet/git-duet-rebase/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"git-duet/internal/cmd"
+	"git-duet/internal/cmdrunner"
+)
+
+func main() {
+	err := cmdrunner.Execute(
+		cmd.New("rebase"))
```

Comment:
Good conversation!

I can see the behavior that @rafecolton describes as being confusing to the user. I think a better solution for this may just be to set `user.name` and `user.email` via `git duet` and `git solo` as described in #4. I'd been putting off implementation of that, but I think the use case above demonstrates why it's advantageous to have that behavior.

Would that suffice, @thewoolleyman ?